src/main/java/edu/cornell/kfs/krad/antivirus/service/impl/ClamAVAntiVirusServiceImpl.java

```diff
@@ -110,18 +110,14 @@ private String readClamAVResponse(Reader socketReader) throws IOException {
             LOG.error("readClamAVResponse, End of stream was reached before reading full ClamAV response!  " +
```

Comment:
since the full response is logged below, is this log statement needed?  Could it be combined?

Reply:
Ok, I'll combine the messages, and will do the same for the "else if" block below.